tests/Auth0.ManagementApi.IntegrationTests/TenantSettingsTests.cs

```diff
@@ -63,7 +63,18 @@ public async Task Test_tenant_settings_sequence()
                 };
 
                 var settingsUpdateResponse = await apiClient.TenantSettings.UpdateAsync(settingsUpdateRequest);
-                settingsUpdateResponse.Should().BeEquivalentTo(settingsUpdateRequest, options => options.Excluding(p => p.SandboxVersion).Excluding(p => p.SandboxVersionsAvailable));
+
+                settingsUpdateResponse.Should().BeEquivalentTo(settingsUpdateRequest, options => options
+                    .Excluding(p => p.Flags)
+                    .Excluding(p => p.SandboxVersion)
+                    .Excluding(p => p.SandboxVersionsAvailable)
+                );
+
+                settingsUpdateResponse.Flags.ChangePwdFlowV1.Should().BeFalse();
+                settingsUpdateResponse.Flags.DisableClickjackProtectionHeaders.Should().BeTrue();
+                settingsUpdateResponse.Flags.EnableAPIsSection.Should().BeTrue();
+                settingsUpdateResponse.Flags.EnableClientConnections.Should().BeFalse();
+                settingsUpdateResponse.Flags.EnablePipeline2.Should().BeTrue();
```

Comment:
Previously we could do a full object comparison, because a lot of properties where not included.
Now, this is not the case anymore as it might be that the values of certain flags are already set to true/false in the Database, while they are not part of the update call here. So the flags we are sent are not identical to the flags we are retrieving.

Example:

Database:
- Flag 1: true
- Flag 2: false

Update call:
- Set Flag 2 to true
- Pass Flag 1 as NULL to not update it

Response: 

Database:
- Flag 1: true
- Flag 2: true